### PR TITLE
none type error

### PR DESCRIPTION
**Pull request by ennozdd**
## Related Issues
fixes: https://github.com/demisto/demisto-sdk/issues/3066

### CODE REVIEW

**Comment by dantavori** [comment]
@ennozdd how did you encounter this?
basically, all mappers should have the internalMapping key (otherwise, what are they mapping?)

**Comment by ennozdd** [comment]
> @ennozdd how did you encounter this? basically, all mappers should have the internalMapping key (otherwise, what are they mapping?)

```
        "dbot_classification_incident_type_all": {
            "dontMapEventToLabels": false,
            "internalMapping": null
        }
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 * Added the PEP484 (no-implicit-optional) hook to **pre-commit**.
+* Fixed an issue where **validate** failed when attempting to map null values in a classifier.
```
Comment (dantavori) [inline CHANGELOG.md:5]:
```suggestion
* Fixed an issue where the content-graph parsing failed on mappers with undefined mapping.
```

**Review by dantavori — APPROVED** [review]
thank you for your contribution 🙏🏽

**Pull request merged**